### PR TITLE
go/worker: add executor livness metrics

### PR DESCRIPTION
**Pull request by ptrus**
Adds liveness (and some other useful) executor prometheus metrics.

TODO:
- [x] test locally (ensure metrics are emitted)

### CODE REVIEW

File: go/worker/common/committee/node.go [hunk 1]
```diff
@@ -62,13 +66,54 @@ var (
...
+		[]string{"runtime"},
+	)
+	livenessTotalRounds = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "oasis_worker_liveness_total_rounds",
+			Help: "Number of total rounds in last epoch.",
+		},
+		[]string{"runtime"},
+	)
+	livenessLiveRounds = prometheus.NewGaugeVec(
```
Comment (kostko) [inline go/worker/common/committee/node.go:97]: Could we also add a percentage (live / total). Maybe it should report 100% if the node is not in the committee just to make alerting easier?

Comment (ptrus) [reply go/worker/common/committee/node.go:97]: Added

**Comment by gw0** [comment]
Awesome, number of peers per runtime is actually what I was hoping for in https://github.com/oasisprotocol/oasis-core/issues/4587.

**Review by kostko — APPROVED** [review]
Nice! This should also be backported to 22.0.x.

File: go/worker/common/committee/node.go [hunk 2]
```diff
@@ -651,6 +708,79 @@ func (n *Node) worker() {
...
+
+	n.logger.Debug("updating periodic worker node metrics")
+
+	epoch := n.Group.GetEpochSnapshot()
+	cmte := epoch.GetExecutorCommittee()
+	if cmte == nil {
+		return
+	}
+
+	executorCommitteePeers.With(labels).Set(float64(len(cmte.Peers)))
```
Comment (kostko) [inline go/worker/common/committee/node.go:732]: Actually this is not the right peers to look at. You probably want to see the list of P2P peers? That one should be in `n.P2P.Peers(runtimeID)`.

Comment (ptrus) [reply go/worker/common/committee/node.go]: updated

**Review by kostko — APPROVED** [review]

**Pull request merged**